src/pages/besquare/index.js

```diff
@@ -16,7 +16,7 @@ import { WithIntl, localize } from 'components/localization'
 const BeSquare = () => (
     <Layout type="be-square">
         <SEO
-            title={localize('BeSquare By Deriv')}
+            title={'BeSquare By Deriv'}
             description={localize(
```

Comment:
should we remove localize in here ?

Reply:
I think it is not required as this page is only marketed in Malaysia.